draft-kuehlewind-taps-crypto-sep.md

```diff
@@ -328,6 +339,20 @@ transition path to enable easy deployment of new protocols.
 
 <!-- CAW: a hint to TLPN could be dropped here -->
 
+# Transport Service Architecture Integration
+
+Pauly et al. {{I-D.pauly-taps-arch}} describe the architecture of a generic
+Transport Services system. Such systems and their APIs provide applications with
+the ability to establish connections for sending and receiving data. The lifetime
+of a connection is comprised of a pre-establishment configuration stage, established
+(connected) stage, and terminated stage. Pre-establishment properties configured
+include: Local and Remote Endpoint, protocol selection properties, and specific
+protocol options. Applications configure security protocols during "preconnection"
```

Comment:
Rather than preconnection, why not just pre-establishment again?

Reply:
Fixed!